### PR TITLE
Public CI runs

### PR DESCRIPTION
**Pull request by tobywf**
*Issue #, if available:* N/A

*Description of changes:* Use a CI runner that doesn't require logging in.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by tobywf** [comment]
Going to add integration step after https://github.com/aws-cloudformation/cloudformation-cli/pull/360 lands.

What do you think of the codecov comments? Does it make sense, since we have 100% coverage?

**Comment by jaymccon** [comment]
> Going to add integration step after [aws-cloudformation/cloudformation-cli#360](https://github.com/aws-cloudformation/cloudformation-cli/pull/360) lands.
> 
> What do you think of the codecov comments? Does it make sense, since we have 100% coverage?

I'd suggest to remove it, will add an additional email per PR to watchers, and with no new info seeing as pre-commit tells us about coverage stats if it's < 100%.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ The CloudFormation Resource Provider Development Kit (RPDK) allows you to author
 
 This plugin library helps to provide runtime bindings for the execution of your providers by CloudFormation.
 
+[![Build Status](https://travis-ci.com/aws-cloudformation/cloudformation-cli-python-plugin.svg?branch=master)](https://travis-ci.com/aws-cloudformation/cloudformation-cli-python-plugin)
```
Comment (jaymccon) [inline README.md]: should add a coverage badge too, 100% is quite impressive 😄

Comment (tobywf) [reply README.md]: i was going to add it, but if we're not doing codecov... maybe we should make our own 🥇

**Comment by tobywf** [comment]
> what are thoughts on auto-publishing to pypi when new commits hit master ?

i'd be happy to consider it once we're happy with the state. i think we'll have to get a bit more sophisticated about tracking versions then though, e.g. somehow embedding the commit hash/tag into the package and then logging it when the CLI runs so we can correlate issues.

**Pull request merged**